meta/acronyms.txt

```diff
@@ -50,22 +54,25 @@ PBS - Peak Burst Size
 PCI - Peripheral Component Interconnect
 PFC - Priority Flow Control
 PIR - Peak Information Rate
-QOS - Quality of Service
```

Comment:
reason why this is removed?

Reply:
metadata checker was complaining about QoS/qos in  saitam.h. I do not see it any more. Will add it back